### ISSUE TITLE
Python 2 End Of Life - (01.20.2020)

### ISSUE THREAD

**Issue by SecureCloud-biz** [arismelachroinos/lscript#236]
This is pasted from the Kali changelog page
https://www.kali.org/releases/kali-linux-2020-1-release/

----------------------

**Python 2 End Of Life**

As a reminder, Python 2 has reached “end of life” on the 1st of January 2020. What this means is, we are removing tools that depend on Python 2. Why? Because they are no longer being maintained, they are not receiving updates, and they need replacing. The pentesting landscape is a dynamic field that is forever changing. It’s time to keep up. We will be doing our best to find alternatives that are actively worked upon.

-------------------

That's one of the issues I have now, both versions of python are installed, but when i install all the wifi tools, some won't run (Python2) tools give a traceback.. This has been happening since early 2019 builds of kali as well as lscript.. the only way I found a way around it was installing them manually & running manually, outside lscript.. which defeats the purpose, c'mon.. the whole idea is to be lazy, is it not? :-)

**Comment by arismelachroinos** [comment]
Hello, there are many things that need to be fixed. This is a difficult period for me as I have many lessons and no free time.

**Comment by SecureCloud-biz** [comment]
Sorry, I added to the comment while you were commenting, please re-read

**Comment by arismelachroinos** [comment]
Yeah also some tools are not really working anymore and have to be removed. There are lots of things that have to be done and I have to check the tools one by one again. I do not have the time this period of time but I am going to do it as soon as I can.

**Comment by SecureCloud-biz** [comment]
Thank You, This is an amazing tool.. (I'm in the process of moving), but I may be able to help with testing, default latest Kali install, (I use LiveCD, wonder if that matters for Py2?), then install lscript and install all tools from within lscript installer, then run each one & let you know which have issues starting & the tools I use, which ones have issues running.. Again thanks for this tool
